posthog/queries/test/test_trends.py

```diff
@@ -2665,6 +2665,61 @@ def test_action_filtering(self):
 
         self.assertEntityResponseEqual(action_response, event_response)
 
+    @snapshot_clickhouse_queries
```

Comment:
should we also get the poe v2 decorator here?

Reply:
given that's not yet merged, I'll do it in a follow up, where I extract the rest of cohort fixes from your PR